.gitignore

```
@@ -20,6 +20,9 @@ settings/*.json
 settings/quickcss/*
 !settings/quickcss/.gitkeep
 
+settings/coremods/*
+!settings/coremods/.gitkeep
```

```suggestion
settings/**/*.json
settings/quickcss/*.css
```
I think this is all you need. Make sure to remove the old line above this and make sure it works.